src/Elsa.CustomActivities/Handlers/Syntax/NestedSyntaxExpressionHandler.cs

```diff
@@ -60,172 +74,60 @@ public NestedSyntaxExpressionHandler(ILogger<NestedSyntaxExpressionHandler> logg
             if (propertyType != null && propertyType == typeof(RadioModel))
             {
                 RadioModel result = new RadioModel();
-                var parsedProperties = ParseToRadioModel(property);
+                var parsedProperties = ParseToList(property);
                 if (parsedProperties != null)
                 {
-                    List<RadioRecord> records = await ElsaPropertiesToRadioRecordList(parsedProperties, evaluator, context);
+                    List<RadioRecord> records = await _radioExpressionHandler.ElsaPropertiesToRadioRecordList(parsedProperties, evaluator, context);
                     result.Choices = records;
                 }
                 return result;
 
             }
-            if (propertyType != null && propertyType == typeof(TextModel))
+            if (propertyType != null && propertyType == typeof(PotScoreRadioModel))
             {
-                TextModel result = new TextModel();
-                var parsedProperties = ParseToTextModel(property);
+                PotScoreRadioModel result = new PotScoreRadioModel();
```

Comment:
same with PotScoreRadioModel - missing test coverage here, based on me running it locally